liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/ServerConfigDocument.java

```diff
@@ -127,6 +127,7 @@ private static void initializeAppsLocation(File serverXML, File configDir, File
                 fProps = parseProperties(new FileInputStream(cfgDirFile));
                 props.putAll(fProps);
             } else if (bootstrapProp != null && !bootstrapProp.isEmpty()) {
+                while (bootstrapProp.values().remove(null));
```

Comment:
I tested it manually - it does. Also, I modified the integration test by adding one more property with null value.